README.md

```diff
@@ -288,11 +229,11 @@ xFormers has a BSD-style license, as found in the [LICENSE](LICENSE) file.
 If you use xFormers in your publication, please cite it by using the following BibTeX entry.
 
 ``` bibtex
-@Misc{xFormers2021,
-  author =       {Benjamin Lefaudeux and Francisco Massa and Diana Liskovich and Wenhan Xiong and Vittorio Caggiano and Sean Naren and Min Xu and Jieru Hu and Marta Tintore and Susan Zhang},
+@Misc{xFormers2022,
+  author =       {Benjamin Lefaudeux and Francisco Massa and Diana Liskovich and Wenhan Xiong and Vittorio Caggiano and Sean Naren and Min Xu and Jieru Hu and Marta Tintore and Susan Zhang and Patrick Labatut and Daniel Haziza},
```

Comment:
feel free to change the author ordering anytime here, I won't be offended personally. Any metrics or semi-reasonable logic works, could be by alphabetical order or contributions for instance, anything which works for you

Reply:
We can keep it like this for now, and update ~every year when we have new contributors